membership-attribute-service/app/models/AccountDetails.scala

```diff
@@ -103,6 +104,7 @@ object AccountDetails {
           "joinDate" -> paymentDetails.startDate,
           "optIn" -> !paymentDetails.pendingCancellation,
           "subscription" -> (paymentMethod ++ Json.obj(
+            "deliveryAddress" -> accountDetails.deliveryAddress,
```

Comment:
please can we expose the Salesforce contact ID aswell as `contactID`? @rBangay can elaborate...